ipmininet/install/build_openr-rc-20190419-11514.sh

```diff
@@ -0,0 +1,301 @@
+#!/bin/bash
+
+set -exo pipefail
+### Install packages for Debian-based OS ###
+
+apt-get update && apt-get install -yq autoconf-archive bison build-essential cmake curl flex git gperf joe libboost-all-dev libcap-dev libdouble-conversion-dev libevent-dev libgflags-dev libgoogle-glog-dev libkrb5-dev libpcre3-dev libpthread-stubs0-dev libnuma-dev libsasl2-dev libsnappy-dev libsqlite3-dev libssl-dev libtool netcat-openbsd pkg-config sudo unzip wget python3-venv python-setuptools python3-setuptools python-pip python3-pip ccache
+apt-get install -yq gcc-'5' g++-'5'
+update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-'5' 40 --slave /usr/bin/g++ g++ /usr/bin/g++-'5'
+update-alternatives --config gcc
+
+PY_MAJOR=${PY_MAJOR:-2}
+
+if [ "$PY_MAJOR" -ne 2 ] && [ "$PY_MAJOR" -ne 3 ]; then
```

Comment:
I would not bother with PY_MAJOR=2, as ipmininet dropped support for python2

Reply:
OpenR uses python afaik for `breeze`, only. `Breeze` is a CLI tool that helps to assert the state of the running OpenR deamon: https://github.com/facebook/openr/blob/master/openr/docs/Breeze.md. Installing `breeze` with Python 2 has no effect on ipmininet. 

However, I guess your suggestion is to remove python 2.x also from "ipmininet-OpenR", right?

Reply:
Yes. If python3 is not available, ipmininet won't run. Arguably this makes
that whole check redundant (in the ipmininet context).

Reply:
Agree. See ae27c71